db/oracle_schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2023_06_12_105944) do
+ActiveRecord::Schema.define(version: 2023_06_12_154444) do
```

Comment:
I think that this needs to be updated to a value after the latest migration which presently is `20230614232715_audited_changes_medium_text.rb`

Reply:
Other than that LGTM, thank you!